dashboard/app/views/levels/_admin.html.haml

```diff
@@ -71,25 +71,28 @@
       - elsif @script_level
         %li= link_to 'edit on levelbuilder', URI.join("https://levelbuilder-studio.code.org/", build_script_level_path(@script_level)).to_s
 
-    -# Include any BubbleChoice parent script_levels in script_level list.
-    - parent_levels = BubbleChoice.parent_levels(@level.name)
-    - parent_script_level_count = parent_levels.flat_map(&:script_levels).count
-    This level is in #{@level.script_levels.count + parent_script_level_count} scripts:
+    This level is in #{@level.script_levels.count} scripts:
     %ul
       - @level.script_levels.each do |script_level|
         %li
           = link_to script_level.script.name, script_level.script
           = "(Lesson ID: #{script_level.stage_id})"
           as
           = link_to build_script_level_path(script_level), build_script_level_path(script_level)
-      - parent_levels.each do |parent_level|
-        - parent_level.script_levels.each do |script_level|
-          %li
-            = link_to script_level.script.name, script_level.script
-            = "(Lesson ID: #{script_level.stage_id})"
-            as
-            - sl_path = build_script_level_path(script_level, sublevel_position: parent_level.sublevel_position(@level))
-            = link_to sl_path, sl_path
+
+    This level is in #{@level.levels_parent_levels.count} other levels:
+    %ul
+      - @level.levels_parent_levels.each do |levels_parent_level|
+        %li
+          %p
+            = "#{levels_parent_level.kind.inspect} in "
+            = link_to levels_parent_level.parent_level.name, levels_parent_level.parent_level
+            = "(position #{levels_parent_level.position})" if levels_parent_level.position.present?
+      %li
+        %p{style: 'width: 400px;'}
+          %small
+            %strong NOTE:
+            This list currently only includes contained levels, project template levels, and BubbleChoice sublevels. Other methods of embedding levels inside other levels are not yet supported.
```

Comment:
It looks like this works for LevelGroup sublevels too:
<img width="445" alt="Screen Shot 2021-07-26 at 4 30 22 PM" src="https://user-images.githubusercontent.com/8001765/127072126-67e92833-49a9-4468-875e-b6b7fcafda90.png">

Reply:
oh, right!

![image](https://user-images.githubusercontent.com/244100/127073361-ed7b44c9-9b36-4410-ab6a-7635b3a4d3e0.png)